src/Repository/AamcPcrsRepository.php

```diff
@@ -119,4 +122,14 @@ protected function attachCriteriaToQueryBuilder(QueryBuilder $qb, $criteria, $or
 
         return $qb;
     }
+
+    /**
+     * @inheritdoc
```

Comment:
I'm inclined to start dropping these `inheritDoc` statements when the method is fully typed and all the inherited doc really has is a comment. Since we're not generating API docs this seems unnecessary.

Reply:
ok.

Reply:
yanked from all Repositories.